src/Jira/Api.php

```diff
@@ -347,6 +348,33 @@ public function getVersions($projectKey)
     }
 
     /**
+     * Helper method to find a specific version based on the name of the version.
+     *
+     * @param string $projectKey Project Key
+     * @param string $name The version name to match on
+     *
+     * @return int|null VersionId on match or null when there is no match
+     */
+    public function findVersionByName($projectKey, $name)
```

Comment:
OK. But in either case until 2.0.0 release all methods not present in 1.x release are subject to removal if such need arises.